monkey/monkey_island/cc/services/node.py

```diff
@@ -41,33 +41,20 @@ def get_displayed_node_by_id(node_id, for_report=False):
             new_node["ip_addresses"] = node["ip_addresses"]
             new_node["domain_name"] = node["domain_name"]
 
-        accessible_from_nodes = []
```

Comment:
This whole file will go away

Reply:
In #2273, yes. I don't think we can remove it right now.